### ISSUE TITLE
ExpiredProviderToken

### ISSUE THREAD

**Issue by Maxim-Chugaev** [node-apn/node-apn#496]
```javascript
const apnProvider = new apn.Provider({
					token: {
						key: './key.p8',
						keyId: '******',
						teamId: '******',
					},
					production: false
				});



setInterval(function () {

	var note = new apn.Notification();
 
	note.expiry = Math.floor(Date.now() / 1000) + 3600; // Expires 1 hour from now. 
	note.badge = 3;
	note.sound = "ping.aiff";
	note.alert = "\uD83D\uDCE7 \u2709 You have a new message";
	note.payload = {'messageFrom': 'John Appleseed'};
	note.topic = "com.example.example-app";


	apnProvider.send(note, '<devicetoken>').then( (result) => {
	    console.log(JSON.stringify(result, null, 2));
	});
}, 1000);
```
result 
```json
{
	"sent": [],
 	"failed": [
 		{
     		"device": "f242e5ad9d45b61f17af6d7bde11e84ace17d6bf90eb9aa94f4a34963e4e5564",
     		"status": "403",
     		"response": {
     			"reason": "ExpiredProviderToken"
     		}
     	}
     ]
}
```
whats wrong?

**Comment by Weijay** [comment]
same here

{ sent: [],
  failed: 
   [ { device: '750A3D35163C0801E28FF979E35F65313093078A721C769096F06ED9CDA09AA6',
       status: '403',
       response: [Object] } ] }

**Comment by zaumreit** [comment]
#480 Please refer this PR.

**Comment by florianreinhart** [comment]
@Maxim-Chugaev this should be fixed in 2.1.3. Can you confirm?

**Comment by Maxim-Chugaev** [comment]
@florianreinhart https://github.com/cmdkoh/node-apn/archive/fix_Getting_app_write_error_%23449_part4.tar.gz"

Solved my error  :D
I have not checked 2.1.3
